dashboard/config/routes.rb

```diff
@@ -59,10 +59,11 @@ module OPS
       post 'log_in'
     end
   end
-  # Temporarily expose Section API routes on dashboardapi, making them
-  # available to pegasus as well.  Also added to api/v1 below.
-  scope 'dashboardapi', module: 'api/v1' do
-    resources :sections, only: [:show, :index]
+  # Section API routes (JSON only)
+  concern :section_api_routes do
+    resources :sections, only: [:show, :index] do
+      resources :students, only: [:index], controller: 'sections_students'
```

Comment:
Here, students is a subresource of sections.  By default this would look for an `Api::V1::StudentsController` but I figure we might want one of those in the future, so I've set an override to use `Api::V1::SectionsStudentsController`.